src/manage_migrators.c

```diff
@@ -1755,12 +1755,19 @@ migrate_209_to_210 ()
       return -1;
     }
 
-  /* Update the database. */
+  /* Do nothing when SQLite is used. During a later
+   * migration the columns will automatically be removed.
+   */ 
+  if (! sql_is_sqlite3 ())
```

Comment:
Should not have a space after the ! but we will catch that when we enable format checking for this file.